api/http/handler/handler.go

```diff
@@ -116,7 +116,7 @@ type Handler struct {
 // @tag.description Manage Portainer settings
 // @tag.name status
 // @tag.description Information about the Portainer instance
-// @tag.name stacks
+// @tag.name docker_stacks
```

Comment:
this isn't right. we also have "kube stacks" there. the description should be corrected

Reply:
previously it duplicates with line 133, same name "stack".

Reply:
right, so I think we can remove this tag (it's not used)